plugins/@grouparoo/airtable/src/lib/appTest.ts

```diff
@@ -0,0 +1,11 @@
+import { TestPluginMethod } from "@grouparoo/core";
+import { IClient } from "./client/interfaces/iClient";
+
+export const appTest: TestPluginMethod<IClient> = async ({ connection }) => {
+  try {
+    const { statusCode, body } = await connection.health();
+    return { success: statusCode === 200, body };
+  } catch (exception) {
+    return { success: false, message: exception.toString() };
```

Comment:
It's a shame that we don't get more info from the API when something goes wrong:
<img width="1312" alt="Screen Shot 2021-12-21 at 2 34 37 PM" src="https://user-images.githubusercontent.com/303226/147006374-24363cbb-4e4a-484e-b7f3-7aa419f3b692.png">